### ISSUE TITLE
TestFest: LiveQuery with TaskFragment of Todo does not work as expected.

### ISSUE THREAD

**Issue by hideki** [couchbase/couchbase-lite-java-core#1602]
During Feb 22nd TestFest, we observed LiveQuery in TaskFragment of Todo app did not work as expected. Peer engineer checked task complete checkbox, and the changes were propagated to Android device. However, it seems LiveQuery indexed new changes once. After that, LiveQuery could reindex with previous revision data. 
Note: this could be caused by multiple LiveQueries are created whenever `Fragment.onCreateView()` method is called. So each Fragment could have 3 to 5 LiveQueries were running.
We need to investigate further.

**Comment by sethrosetter** [comment]
After the trying to repro today with 7 android device for about an hour, we were unable to hit this issues. We were using the new app LiveQuery changes.

**Comment by sethrosetter** [comment]
@hideki Can we push to 1.4.1?

**Comment by hideki** [comment]
@sethrosetter We could retest this scenario for 1.4.1 release. Beside retesting, I don't know what we could do for this issue?

**Comment by sethrosetter** [comment]
@hideki I am in agreement. Just want to get @djpongh's approval

**Comment by djpongh** [comment]
Agreed. Potential candidate for 1.4.1, although I don't really want to have a 1.4.1 for CBL.
